bookwyrm/templates/book/format_info.html

```diff
@@ -0,0 +1,39 @@
+{% spaceless %}
+
+{% load i18n %}
+{% load book_display_tags %}
+
+{% firstof book.physical_format_detail book.get_physical_format_display as format %}
+{% firstof book.physical_format book.physical_format_detail as format_property %}
+{% with pages=book.pages %}
+{% if format or pages %}
+
+{% if format_property %}
+<meta itemprop="bookFormat" content="{{ format_property }}">
+{% if book.physical_format == "AudiobookFormat" and book.audiobook_play_time %}
+    <meta itemprop="timeRequired" content="{{ book.audiobook_play_time|iso_duration }}">
```

Comment:
When I tested this, it looks like this line was passing in a string-ified version of the duration (for the arbitrary numbers I tried, `85 days, 17:36:00`), which was throwing an error in the filter. I think that this is because I entered a big enough number that it went over hours and into days